<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMatrix2</class>
 <widget class="QDialog" name="DialogMatrix2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxRows1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxColumns1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRows2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxColumns2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table1Widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>351</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table2Widget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>371</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getData1Button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>48</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>get data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getData2Button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>get data</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table3Widget">
   <property name="geometry">
    <rect>
     <x>935</x>
     <y>111</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>150</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subtractButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplyButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>368</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value1Edit">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>480</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fill1Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>fill with value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fill2Button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>fill with value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value2Edit">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>480</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>882</x>
     <y>280</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>49</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Result Matrix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>479</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a value to fill with</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a value to fill with</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
